jp/0287901.json

```diff
@@ -1 +1 @@
-{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第８地割～第１４地割（玉川、"}
\ No newline at end of file
+{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第８地割～第１４地割"}
```

Comment:
@sunfmin Seems it's not consistently formatted like "yes (no)". This should probably be: `"town": "種市"`

Reply:
```
FeliMac:zipcode sunfmin$ rg ～
japanpost/main.go
101:		// remove （１～４丁目）

jp/4280049.json
1:{"prefecture":"静岡県","city":"島田市","town":"２７６、２９４～３００、３０２～３"}

jp/0287911.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第３９地割～第４５地割"}

jp/6060068.json
1:{"prefecture":"京都府","city":"京都市左京区","town":"～１８、２０～２２、２４～２６、３"}

jp/0295501.json
1:{"prefecture":"岩手県","city":"和賀郡西和賀町","town":"下左草７７地割～下左草８０地割"}

jp/0295521.json
1:{"prefecture":"岩手県","city":"和賀郡西和賀町","town":"小繋沢５４地割～小繋沢５６地割"}

jp/0295506.json
1:{"prefecture":"岩手県","city":"和賀郡西和賀町","town":"湯之沢３１地割～湯之沢３５地割"}

jp/5970045.json
1:{"prefecture":"大阪府","city":"貝塚市","town":"－４、９３４～９６８、１０１３－１"}

jp/0287901.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第８地割～第１４地割"}

jp/0287917.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第５０地割～第７０地割"}

jp/9910801.json
1:{"prefecture":"山形県","city":"西村山郡大江町","town":"２２５７－２８、２３１６～２３１８"}

jp/0613774.json
1:{"prefecture":"北海道","city":"石狩郡当別町","town":"－４、５４０７－５、５４４５～５４"}

jp/0287916.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第７１地割～第７４地割"}

jp/0287915.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第１５地割～第２１地割"}

jp/0287903.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第１地割～第３地割"}

jp/0295504.json
1:{"prefecture":"岩手県","city":"和賀郡西和賀町","town":"槻沢２５地割～槻沢２８地割"}

jp/8910304.json
1:{"prefecture":"鹿児島県","city":"指宿市","town":"１０６０３～１１０１５）"}

jp/0287918.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第４６地割～第４９地割"}

jp/2892605.json
1:{"prefecture":"千葉県","city":"旭市","town":"３５９８～３６１４番地）"}

jp/3250304.json
1:{"prefecture":"栃木県","city":"那須郡那須町","town":"６１１４～６１４１、６３２８、６４"}

jp/3994232.json
1:{"prefecture":"長野県","city":"駒ヶ根市","town":"、２４３３～２４５０）"}

jp/0287902.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第４地割～第７地割"}

jp/0287914.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第２２地割～第２３地割"}

jp/0295522.json
1:{"prefecture":"岩手県","city":"和賀郡西和賀町","town":"野々宿６０地割～野々宿６２地割"}

jp/5900158.json
1:{"prefecture":"大阪府","city":"堺市南区","town":"７３３－１～７３３－２、７３５、７"}

jp/0287913.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第２４地割～第２５地割"}

jp/0295503.json
1:{"prefecture":"岩手県","city":"和賀郡西和賀町","town":"湯田１９地割～湯田２１地割"}

jp/0287912.json
1:{"prefecture":"岩手県","city":"九戸郡洋野町","town":"種市第２６地割～第３８地割"}
```

Very not consistant... clear the town value when ever it has `～` in it?

Reply:
```

Reply:
Is this *all* of the postcodes with `~` in `town`? If so, they can be fixed by hand I think?

Reply:
it's generated from their csv, if by hand, next time we update the data, it will be overwritten.

Reply:
Yes, so don't edit the JSON directly. To avoid it you keep a manual correction list, basically something like

```
corrections := map[string]string{
  "種市第４地割～第７地割": "種市",
  "３５９８～３６１４番地）": ""
  "湯田１９地割～湯田２１地割": "湯田"
}
```

(you could include the zip code or prefecture or city, in the mapping, to make it more specific).

Then when you generate the files, you can also print out mappings that seem to be unused, so when Japan Post changes the data, the old mappings can be deleted.

It would be nice to keep the original text to display as a hint on the form :trollface: